tonuino_cards_manager/main.py

```diff
@@ -78,7 +81,12 @@ def main():
             extra2=card.extra2,
         )
 
-        qrdata.append(f"{card_bytecode};{card.create_carddesc(cardno)}")
+        card_description = card.create_carddesc(cardno)
+        qrdata.append(f"{card_bytecode};{card_description}")
+        card_description = card_description.split("(")
+        card_description = card_description[1].split(")")
+        audiolength_str = str(timedelta(seconds=sum(audiolength)))
+        mylist.append([cardno, card_description[0], len(audiolength), audiolength_str])
```

Comment:
Comments, and suggestions for more clear variable names

```suggestion
        card_description = card.create_carddesc(cardno)
        # Add card to QR code generation
        qrdata.append(f"{card_bytecode};{card_description}")
        # Extract content of card
        card_description = card_description.split("(")
        card_description = card_description[1].split(")")
        card_total_audio_length = str(timedelta(seconds=sum(card_audiolengths)))
        mylist.append([cardno, card_description[0], len(card_audiolengths), card_total_audio_length])
```